stdlib/public/core/Span/RawSpan.swift

```diff
@@ -0,0 +1,654 @@
+  /// above may produce an invalid value of `T`.
+  ///
+  /// - Parameters:
+  ///   - offset: The offset from this pointer, in bytes. `offset` must be
+  ///     nonnegative. The default is zero.
+  ///   - type: The type of the instance to create.
+  /// - Returns: A new instance of type `T`, read from the raw bytes at
+  ///     `offset`. The returned instance isn't associated
+  ///     with the value in the range of memory referenced by this pointer.
+  @_disallowFeatureSuppression(NonescapableTypes)
+  @unsafe
+  @_alwaysEmitIntoClient
+  public func unsafeLoadUnaligned<T: BitwiseCopyable>(
+    fromByteOffset offset: Int = 0, as: T.Type
+  ) -> T {
+    _precondition(
+      UInt(bitPattern: offset) <= UInt(bitPattern: _count) &&
+      MemoryLayout<T>.size <= (_count &- offset),
+      "byte offset range out of bounds"
+    )
+    return unsafeLoadUnaligned(fromUncheckedByteOffset: offset, as: T.self)
+  }
+
+  /// Returns a new instance of the given type, constructed from the raw memory
+  /// at the specified offset.
+  ///
+  /// The memory at this pointer plus `offset` must be initialized to `T`
+  /// or another type that is layout compatible with `T`.
+  ///
+  /// This is an unsafe operation. This function does not validate the bounds
+  /// of the memory access, and failure to meet the preconditions
+  /// above may produce an invalid value of `T`.
+  ///
+  /// - Parameters:
+  ///   - offset: The offset from this pointer, in bytes. `offset` must be
+  ///     nonnegative. The default is zero.
+  ///   - type: The type of the instance to create.
+  /// - Returns: A new instance of type `T`, read from the raw bytes at
+  ///     `offset`. The returned instance isn't associated
+  ///     with the value in the range of memory referenced by this pointer.
+  @_disallowFeatureSuppression(NonescapableTypes)
+  @unsafe
+  @_alwaysEmitIntoClient
+  public func unsafeLoadUnaligned<T: BitwiseCopyable>(
+    fromUncheckedByteOffset offset: Int, as: T.Type
+  ) -> T {
+    _start().loadUnaligned(fromByteOffset: offset, as: T.self)
+  }
+}
+
+@_disallowFeatureSuppression(NonescapableTypes)
+@available(SwiftStdlib 6.1, *)
+extension RawSpan {
+  /// Returns a Boolean value indicating whether two `RawSpan` instances
+  /// refer to the same region in memory.
+  @_disallowFeatureSuppression(NonescapableTypes)
+  @_alwaysEmitIntoClient
+  public func isIdentical(to other: Self) -> Bool {
+    (self._pointer == other._pointer) && (self._count == other._count)
+  }
+
+  /// Returns the offsets where the memory of `span` is located within
+  /// the memory represented by `self`
+  ///
+  /// Note: `span` must be a subrange of `self`
+  ///
+  /// Parameters:
+  /// - span: a subrange of `self`
+  /// Returns: A range of offsets within `self`
+  @_disallowFeatureSuppression(NonescapableTypes)
+  @_alwaysEmitIntoClient
+  public func byteOffsets(of span: borrowing Self) -> Range<Int>? {
```

Comment:
```suggestion
  public func byteOffsets(of other: borrowing Self) -> Range<Int>? {
```

(With references updated etc.)